entrypoint.sh

```diff
@@ -10,18 +10,26 @@ if [[ -z "${PROXY_LIST_URL}" ]]; then
     echo "Using mounted proxy list"
     touch $BASE_DIR/proxy-list.txt
     echo " ---> Done"
-else
+elif [[ -z "${BRIGHT_DATA_API_TOKEN}"]]; then
     echo "Downloading proxy list from $PROXY_LIST_URL"
     curl -s $PROXY_LIST_URL > $BASE_DIR/proxy-list.txt
     echo " ---> Done"
+else
+    echo "Downloading proxy list from $PROXY_LIST_URL"
```

Comment:
It's not downloading from this list, right?

Reply:
It is 🤔 why do you say it isn't?

Reply:
It's a bit odd, I would think the API address would be hardcoded and not a ENV var, it's not a secret, it's not supposed to change 🤷🏼

Reply:
It changes because it includes the zone.. and I wanted to keep the same condition to go from mounted proxy list to the downloaded one.
Otherwise I'd have to create an ENV VAR just to say if it should be one or the other

Reply:
but you're already using the bright data API token for that, so I don't see the difference 🤔

Reply:
Changed